tox.ini

```diff
@@ -1,30 +1,60 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
-envlist = flake8,
-          py27,
-          py35,
-          py36,
-          py37,
-          py38,
-          pypy,
-          pypy3,
-          coverage
+envlist =
+    lint,
+    py27,
+    pypy,
+    py35,
+    py36,
+    py37,
+    py38,
+    pypy3,
+    coverage
 
 [testenv]
 usedevelop = true
+deps =
+    zope.testrunner
 commands =
     zope-testrunner --test-path=src []
 extras = test
 
-[testenv:coverage]
-basepython = python3.7
+[testenv:lint]
+basepython = python3
+skip_install = true
+deps =
+    flake8
+    check-manifest
+    check-python-versions
 commands =
-    coverage run -m zope.testrunner --test-path=src []
-    coverage report --fail-under=86
+    flake8 src setup.py
+    check-manifest
+    check-python-versions .
```

Comment:
You seem not to use the latest version of the script, I eliminated the dot at the end of is line as it is not necessary.

Reply:
Actually, I applied an earlier version, and now I applied the current version, and tox.ini does not get updated. So I fixed it manually.

Reply:
Ok, the pull failed because of the changed `packages.txt`...